From 507d8e8d4c60fa6175d327c4f2dac307dc11ccd6 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Fri, 19 Sep 2014 16:51:34 +0200
Subject: [PATCH 3/6] libparted: also link to UUID_LIBS

* libparted/Makefile.am (libparted_la_LIBADD): Add UUID_LIBS.
This fixes building parted with the gold linker.
---
 libparted/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libparted/Makefile.am b/libparted/Makefile.am
index edc5f2e..cb58648 100644
--- a/libparted/Makefile.am
+++ b/libparted/Makefile.am
@@ -58,6 +58,7 @@ libparted_la_LIBADD =	\
   $(DM_LIBS)		\
   $(SELINUX_LIBS)	\
   $(LIB_BLKID)		\
+  $(UUID_LIBS)		\
   $(INTLLIBS)
 
 EXTRA_DIST	      = mbr.s
-- 
1.9.3

